app/assets/javascripts/discourse/templates/components/group-selector.hbs

```diff
@@ -1 +0,0 @@
-<input class='ember-text-field group-names' type="text" placeholder={{placeholder}} name="groups">
```

Comment:
I will keep the template since it already exists plus it is easier to add stuff to a template than a StringBuffer